Packages/com.unity.inputsystem/Documentation~/Concepts.md

```diff
@@ -1,37 +1,18 @@
-
-|Concept|Description|
-|-------|-----------|
-|[**Processor**](Processors.md)|An operation that the Input System applies to an input value. For example, an "invert" Processor inverts a floating-point value.|
-|[**Phase**](Interactions.md#operation)|The current state of an [Interaction](Interactions.md), for example, "Started", "Performed", or "Canceled".|
-|[**Control Scheme**](ActionBindings.md#control-schemes)|Allows you to define mappings of Bindings to different Control Schemes, and to switch your Action Maps between different Control Schemes to enable different subsets of Bindings for your Actions. Control Schemes can have associated Device types, so that the game can automatically enable them for [users](UserManagement.md) when using that type of Device.|
-|[**Action Map**](ActionAssets.md#editing-action-maps)|A named collection of Actions. You can simultaneously enable or disable all Actions in an action map, so it is useful to group Actions in Action Maps by the context in which they are relevant (for example: "gameplay").|
+|[**Control**](Controls.html)|The separate individual parts of an input device which each send input values into Unity. For example, a gamepad’s **controls** comprise multiple buttons, sticks and triggers, and a mouse’s controls include the two X and Y sensors on the underside, and the various buttons and scroll wheels on the top side.|
+|[**Action**](Actions.html)| Actions are a high-level concept that describe individual things that a user might want to do in your game or app, such as "Jump" within a game, or "Select" in an on-screen UI. They are things a user can do in your game or app as a result of input, regardless of what device or control they use to perform it. Actions generally have conceptual names that you choose to suit your project, and should usually be verbs. For example "Run", "Jump" "Crouch", "Use", "Start", "Quit".|
+|**Action Map**| Action Maps allow you to organise Actions into groups which represent specific situations where a set of actions make sense together. You can simultaneously enable or disable all Actions in an action map, so it is useful to group Actions in Action Maps by the context in which they are relevant. For example, you might have one action map for controlling a player, and another for interacting with your game's UI.|
+|[**Binding**](ActionBindings.html)| A connection defined between an **Action** and specific device controls. For example, your "Move" action might have bindings to the arrow keys and WSAD keys on the keyboard, and the left stick on a joypad, and the primary 2D axis on a VR controller. Multiple bindings like this means your game can accept cross-platform input. |
+|**Action Reference**| A reference in your script to an Input Action. Once you have a reference to an action, you can either read the current value or state of the action (also known as "polling"), or set up a callback to call your own method when actions are performed.|
```

Comment:
Relating to "Action Reference", I notice we call this "polling" which is incorrect in this case. In one sense it could be perceived as such since you read the current value each frame so maybe o.k. but its not going to the device and fetching the latest value under the hood, it is only returning the value that was "polled" during InputSystem update.